<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreprocessingOptions</class>
 <widget class="QWidget" name="PreprocessingOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure toolchain</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Binarization method:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="bin_algoCbox"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="subsampleCb">
     <property name="text">
      <string>Run on subsampled image</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="remove_bgCb">
     <property name="text">
      <string>Remove background (slow)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="deskewCb">
     <property name="text">
      <string>Deskew</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="remove_noiseCb">
     <property name="text">
      <string>Remove noise</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>48</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
